x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/step_logistics.tsx

```diff
@@ -625,6 +626,41 @@ export const RestoreSnapshotStepLogistics: React.FunctionComponent<StepProps> =
           />
         </EuiFormRow>
       </EuiDescribedFormGroup>
+
+      {/* Include aliases */}
+      <EuiDescribedFormGroup
+        title={
+          <EuiTitle size="s">
+            <h3>
+              <FormattedMessage
+                id="xpack.snapshotRestore.restoreForm.stepLogistics.includeAliasesTitle"
+                defaultMessage="Include aliases"
```

Comment:
The current copy is fine, but I think leading with `Restore ...` sounds a bit more direct and seems more aligned with other options on this page.
```suggestion
                defaultMessage="Restore aliases"
```